python/proton_bremsstrahlung.py

```diff
@@ -188,7 +188,7 @@ def hProdPDF(mDarkPhoton, epsilon, norm, binsp, binstheta, tmin = -0.5 * math.pi
             hPDF.Fill(p,theta,w)
             hPDFtheta.Fill(theta,w)
             hPDFp.Fill(p,w)
-    hPdfFilename = sys.modules['__main__'].outputDir+"/ParaPhoton_eps%s_m%s%s.root"%(epsilon,mDarkPhoton,suffix)
```

Comment:
This isn't particularly elegant, but appreciate the attempt to not hard-code the location.

Reply:
I have seen elegant ones in somewhere at the FairShip.. Maybe it was in the run_fixedTarget.py of Thomas.
I will have a look at it.